cpp/src/neighbors/detail/nn_descent.cuh

```diff
@@ -1196,9 +1196,16 @@ void GNND<Data_t, Index_t>::build(Data_t* data,
     raft::matrix::slice<DistData_t, int64_t, raft::row_major>(
       res, raft::make_const_mdspan(graph_d_dists.view()), output_dist_view, coords);
 
+    // distance post-processing
     if (build_config_.metric == cuvs::distance::DistanceType::L2SqrtExpanded) {
       raft::linalg::map(
         res, output_dist_view, raft::sqrt_op{}, raft::make_const_mdspan(output_dist_view));
+    } else if (build_config_.metric == cuvs::distance::DistanceType::InnerProduct) {
```

Comment:
nitpick: Can we use the `is_min_close` function here instead of checking for IP distance itself?
```suggestion
    } else if (!cuvs::distance::is_min_close(build_config_.metric)) {
```

Reply:
+1